code/ATMOSPHERICS/hvac/spaceheater.dm

```diff
@@ -21,8 +21,32 @@
 	flags = FPRINT
 	machine_flags = SCREWTOGGLE
 
+/obj/machinery/space_heater/campfire
+	name = "campfire"
+	icon_state = "campfire0"
+	base_state = "campfire"
+	desc = "Warning: May attract Space Bears."
+	light_power_on = 1.5
```

Comment:
I hope you have tested this. 2 seems better, 3 even since it's an open fire. Also, please use fire color